### ISSUE TITLE
[Feature request] Enable center column in tableColumnExtensions

### ISSUE THREAD

**Issue by felansu** [DevExpress/devextreme-reactive#791]
- [x] I have searched this repository's [issues](https://github.com/devexpress/devextreme-reactive/issues) and believe that this is not a duplicate.

### Current Behaviour
Ref: https://github.com/DevExpress/devextreme-reactive/issues/765

Can't center/middle-align.

### Expected Behaviour

In [documentation](https://devexpress.github.io/devextreme-reactive/react/grid/docs/reference/table/#tablecolumnextension) don't exist center align. I would like to be able to center-align column. Example:
```
 tableColumnExtensions: [
    {columnName: 'someColumn', align: 'center', width: 90},
  ],
```

### Steps to Reproduce (for Bugs)

Not a bug, a feature request.

### Your Environment

"@devexpress/dx-react-core": "1.1.0-beta.1",
"@devexpress/dx-react-grid": "1.1.0-beta.1",
"@devexpress/dx-react-grid-material-ui": "1.1.0-beta.1",
"react": "v16.2.0",
"material-ui": "v1.0.0-beta.34",

**Comment by SergeyAlexeev** [comment]
Hi, thank you for your suggestion! We will discuss the possibility of implementing such a functionality in the future.

**Comment by MaximKudriavtsev** [comment]
We've released the [1.1.0-beta.3](https://github.com/DevExpress/devextreme-reactive/releases/tag/v1.1.0-beta.3) version containing described changes.

You can install it with the `next` tag:

`npm i --save @devexpress/dx-react-core@next @devexpress/dx-react-grid@next`
`npm i --save @devexpress/dx-react-grid-bootstrap3@next`
`npm i --save @devexpress/dx-react-grid-bootstrap4@next`
`npm i --save @devexpress/dx-react-grid-material-ui@next`

**Comment by felansu** [comment]
Thanks !
